src/Service/PdfCreatorInterface.php

```diff
@@ -8,5 +8,5 @@ interface PdfCreatorInterface
 {
     public function create(): void;
 
-    public function generatePdf(string $template, string $name): string;
+    public function generatePdf(string $template, string $name): void;
```

Comment:
Where do you get the PDF back? We're aiming to get the PDF as a string so that we can either stream it back to the user or easily store it somewhere using any sort of filesystem abstraction layer.